code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -178,7 +178,7 @@
 	icon_state = "stimpakpen"
 	volume = 10
 	amount_per_transfer_from_this = 10
-	list_reagents = list(/datum/reagent/medicine/stimpak = 10)
+	list_reagents = list(/datum/reagent/medicine/stimpak = 10, /datum/reagent/medicine/stimulants = 10, /datum/reagent/medicine/omnizine = 10)
```

Comment:
i think that aforementioned PR also removes omnizine.. probably be rid of this file's changes wholesale since they undo a lot of their work